.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: publish
+
+on:
+  push:
+    paths:
+      - "src/**"
+      - ".github/workflows/publish.yml"
+    branches:
+      - main
+  pull_request:
+    paths:
+      - "src/**"
+      - ".github/workflows/publish.yml"
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        machine: [windows-latest, macos-latest, ubuntu-20.04]
```

Comment:
do we need both this line and the `machine` in `include`? maybe this one can go?

Reply:
How I understand the include part to work is that if the key value pairs in the matrix match the combinations in include the correct platform value is added to the matrix. 

So I think if we removed the `machine` under `matrix` it would always choose the first entry in `include` which is windows

https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations

I think if we ever go back and look at arm64 we could use `include` for extra parameters at the end like is required for linux arm64 building. If we wanted to get rid of `include` I think the matrix supports passing it arrays https://github.com/orgs/community/discussions/24981

Reply:
> So I think if we removed the `machine` under `matrix` it would always choose the first entry in include which is windows

I don't think this is true. If there's no match, objects in `include` are all added to the list. From the docs:

> For each object in the `include` list, the key:value pairs in the object will be added to each of the matrix combinations if none of the key:value pairs overwrite any of the original matrix values.

We have quite a few examples of having no parameter directly under `matrix` and relying purely on `include`, and all objects in `include` are run.

Reply:
Oh yea I see in slims. You're right then I think we are good to remove the line 19